src/query/storages/fuse/src/operations/common/processors/sink_commit.rs

```diff
@@ -117,9 +124,19 @@ where F: SnapshotGenerator + Send + 'static
             input,
             need_lock,
             start_time: Instant::now(),
+            prev_snapshot_id,
```

Comment:
What about move the snapshot_id into `ConflictResolveContext`: https://github.com/datafuselabs/databend/blob/282cbc25b8009a66d0dab37d0917bb5322b61db2/src/query/storages/fuse/src/operations/common/snapshot_generator.rs#L332
So that if we change the conflict resolve method(i.e. int64 -> int32 may be resovable) later , we can reuse the retry process：https://github.com/datafuselabs/databend/blob/282cbc25b8009a66d0dab37d0917bb5322b61db2/src/query/storages/fuse/src/operations/common/processors/sink_commit.rs#L213

Reply:
Let's fix it when problem actually occurs